lib/active_merchant/billing/network_tokenization_credit_card.rb

```diff
@@ -27,6 +27,10 @@ def source
         end
       end
 
+      def credit_card?
+        true
```

Comment:
I _think_ this should be `false` as the old call of `is_a?(CreditCard)` would have failed if the class was a `NetworkTokenizationCreditCard`.